music_assistant/constants.py

```diff
@@ -301,15 +305,57 @@
     depends_on=CONF_FLOW_MODE,
 )
 
+CONF_ENTRY_SMART_FADES_MODE = ConfigEntry(
+    key=CONF_SMART_FADES_MODE,
+    type=ConfigEntryType.STRING,
+    label="Smart Fades Mode",
+    options=[
+        # TODO: Question: Should we move the enum to models and import it from there ?
+        ConfigValueOption("Disabled", "disabled"),
+        ConfigValueOption("Smart Fades", "smart_fades"),
+        ConfigValueOption("Default Crossfade", "default_crossfade"),
```

Comment:
```suggestion
        ConfigValueOption("Standard Crossfade", "default_crossfade"),
```

i would use "regular" or "standard" here instead of the word default